Flow.Launcher/Flow.Launcher.csproj

```diff
@@ -77,6 +77,24 @@
       <SubType>Designer</SubType>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <COMReference Include="SHDocVw">
+      <VersionMinor>1</VersionMinor>
+      <VersionMajor>1</VersionMajor>
+      <Guid>eab22ac0-30c1-11cf-a7eb-0000c05bae0b</Guid>
+      <Lcid>0</Lcid>
+      <WrapperTool>tlbimp</WrapperTool>
+      <Isolated>false</Isolated>
+      <EmbedInteropTypes>true</EmbedInteropTypes>
+    </COMReference>
+    <COMReference Include="Shell32">
+      <VersionMinor>0</VersionMinor>
+      <VersionMajor>1</VersionMajor>
+      <Guid>50a7e9b0-70ef-11d1-b75a-00a0c90564fe</Guid>
+      <Lcid>0</Lcid>
+      <WrapperTool>tlbimp</WrapperTool>
+      <Isolated>false</Isolated>
+      <EmbedInteropTypes>true</EmbedInteropTypes>
+    </COMReference>
```

Comment:
What do you think about the COMREference? Is it better to use dynamic typing as what you did in the runner plugin? Somehow the dotnet cli doesn't support this feature, and the previous workaround is to copy the reference dll and directly reference that dll.

Reply:
I think if the COMReference works without any issues, then that's pretty nice. But I honestly have no clue.

Reply:
I would say since you have shared a way to prevent usage of comreference, let's use that. Previously I workaround this by copying out the dll and reference that directly.

Reply:
Though I personally think it may be better to have strongly type?

Reply:
https://github.com/dotnet/msbuild/issues/3986

Reply:
Apparently one can define the COM import interfaces themselves. Like this app does https://github.com/emoacht/Monitorian/blob/c16345d1aa679ca463a14b4c1ef0917a524fe31a/Source/Monitorian.Core/Models/Monitor/LightSensor.cs , or like how the .NET framework does it https://referencesource.microsoft.com/#PresentationFramework/src/Framework/MS/Internal/AppModel/ShellProvider.cs,3ec518c827af4fd7

Reply:
yeah for sure, but I think that's pretty messy. We do have some example about that.